srcpkgs/r2ghidra-dec/template

```diff
@@ -0,0 +1,36 @@
+# Template file for 'r2ghidra-dec'
+_cutterversion=1.11.1
+_ghidracommit=6c10f36f06468f866188cccf960c019779fb9028
+_pugixmlcommit=fdf02957533645069bbea7cd2ed85a3014b42cf3
+pkgname=r2ghidra-dec
+version=4.5.0
+revision=1
+archs="x86_64"
+build_style=cmake
+configure_args="-DCMAKE_INSTALL_PREFIX=/usr -DBUILD_CUTTER_PLUGIN=ON -DCUTTER_SOURCE_DIR=${XBPS_BUILDDIR}/cutter-${_cutterversion}"
```

Comment:
`-DCMAKE_INSTALL_PREFIX=/usr ` is unneeded, specified in the `cmake` build_style. Also split across two lines, line length should be only 80 characters.